### DISCUSSION TITLE
in ./node_modules/@formily/antd/node_modules/react-sticky-box/dist/index.js

### DISCUSSION THREAD

**Discussion by robin536180** [alibaba/formily#3215] [Q&A 2.x]
npm start 报错
Module not found: Error: Can't resolve '/Desktop/ui/node_modules/@umijs/babel-preset-umi/node_modules/@babel/runtime/helpers/esm/slicedToArray' in '/Desktop/ui/node_modules/@formily/antd/node_modules/react-sticky-box/dist'Did you mean 'slicedToArray.js'?
BREAKING CHANGE: The request '/Desktop/ui/node_modules/@umijs/babel-preset-umi/node_modules/@babel/runtime/helpers/esm/slicedToArray' failed to resolve only because it was resolved as fully specified
(probably because the origin is a '*.mjs' file or a '*.js' file where the package.json contains '"type": "module"').
The extension in the request is mandatory for it to be fully specified.
Add the extension to the request.
package.json
<img width="380" alt="截屏2022-06-20 下午11 05 38" src="https://user-images.githubusercontent.com/8565176/174631786-ac00c228-efae-44b0-a003-f7924ea9f401.png">
yarn.lock
<img width="881" alt="截屏2022-06-20 下午11 06 26" src="https://user-images.githubusercontent.com/8565176/174631855-f9304816-0a51-4ad3-b59f-feaa3f69ddae.png">

**Comment by silvmind** [comment]
更新下`@babel/runtime-corejs3`依赖包
修改.umirc.js配置 `nodeModulesTransform: { type: 'none' }`
我出现了同样问题，升级了这个依赖包后正常
